examples/oneapi/dpc/source/linear_kernel/linear_kernel_dense_batch.cpp

```diff
@@ -66,6 +66,9 @@ void run(sycl::queue &queue) {
 
 int main(int argc, char const *argv[]) {
     for (auto device : list_devices()) {
+        std::cout << "Running on "
```

Comment:
You need to remove https://github.com/oneapi-src/oneDAL/pull/713/files#diff-a512b7a5e9f6e5cd2a00d4f117b11efaL27